src/python/WMComponent/RucioInjector/RucioInjectorPoller.py

```diff
@@ -465,6 +466,9 @@ def insertContainerRules(self):
             else:
                 # then it's a T0 container placement
                 ruleKwargs['priority'] = 4
+                if not rseName.endswith("_Tape"):
```

Comment:
should we also make sure that the `lifetime` is greater than 0 (thus, not using the database default value)?

Reply:
Hi Alan I don't know if I understood your suggestion but this is solved from T0 code in this [line](https://github.com/dmwm/T0/blob/15ab4669bb6830dfc9043b810e33a72657ec7ea0/src/python/T0/RunConfig/Tier0Config.py#L551) with a default value.